### PR TITLE
Tweaked font sizes and whitespace + added page resize handler

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#8245

*Description of changes:*
- tweaked font sizes
- tweaked whitespace
- added page resize handler

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: client/src/components/application/Transactions.vue [hunk 1]
```diff
@@ -62,7 +62,7 @@
         </template>
 
         <v-layout id="transaction-title-bar" text-xs-center>
-          <v-flex class="py-2 fs-18 fw-700">TRANSACTION HISTORY</v-flex>
+          <v-flex class="py-2 fs-16 fw-700">TRANSACTION HISTORY</v-flex>
```
Comment (cameron-freshworks) [inline client/src/components/application/Transactions.vue:65]: Super minor but will cause a conflict. Maybe just easier to leave this out of your pr?

**Review by cameron-freshworks — APPROVED** [review]
Looks good!

File: client/src/components/application/Transactions.vue (hunk 1 above)
Comment (severinbeauvais) [reply client/src/components/application/Transactions.vue]: Done. Please remember to make this change!

Also, lmk when your PR is merged and I can build/deploy this project to Dev.

Comment (cameron-freshworks) [reply client/src/components/application/Transactions.vue]: Thanks, absolutely, will do 👍

**Pull request merged**